Pipeline/Process_Data.m

```diff
@@ -46,16 +46,17 @@ function Process_Data(fname,folder,ForceFlage,ForceComFlage,ForceplateNum)
   GRFdata =  [GRFdata [data.fp_data.GRF_data(fp_Number(i)).P*RMatrix]/p_sc];
   GRFdata =  [GRFdata [data.fp_data.GRF_data(fp_Number(i)).M*RMatrix]/p_sc];
 end
+
 %Separates ground reaction forces onto each foot
     if ForceComFlage
-        sGRFdata=SeparateGRF(MarkerData,GRFdata,Markerset);
+        sGRFdata=TM_SeparateGRF(MarkerData,GRFdata,Markerset);
```

Comment:
I have sent a commit on your pull to make this part compatible with rest of code.